test/unit/mdc-dialog/foundation.test.js

```diff
@@ -394,7 +393,11 @@ test(`interaction does nothing when ${strings.ACTION_ATTRIBUTE} attribute is not
   INTERACTION_EVENTS.forEach((event) => {
```

Comment:
Let's separate this test case for each interaction callback (click & keydown) since the callback methods are separated.